### DISCUSSION TITLE
Question regarding an animation with a scrollbar

### DISCUSSION THREAD

**Discussion by JanJohn123** [google/model-viewer#4054] [Q&A]
I have 5 3D models and would like to 'animate' them, I would need a scrollbar from 1 to 5, when you scroll to 1, model no. 1 is seen, when to 2, model 2 is seen etc. Is it possible with your viewer? Thanks a lot.

**Comment by elalish** [comment]
yes: https://modelviewer.dev/examples/augmentedreality/index.html#webXR
